proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -172,6 +172,7 @@ message TransferTaskInfo {
     // (-- api-linter: core::0140::prepositions=disabled
     //     aip.dev/not-precedent: "after" is used to indicate sequence of actions. --)
     bool delete_after_close = 15;
+    int64 close_transfer_task_id = 16;
```

Comment:
You don't need this one in transfer deletion task. This data is accessible in the mutable state in transfer task

Reply:
Done